platform/platform.go

```diff
@@ -60,37 +59,8 @@ func getPlatformInfo() (platformInfo map[string]string, warnings []string, err e
 	}
 
 	platformInfo["goV"] = strings.Replace(runtime.Version(), "go", "", -1)
-	// If this errors, swallow the error.
-	// It will usually mean that Python is not on the PATH
-	// and we don't care about that.
-	pythonV, e := getPythonVersion(exec.Command)
-
-	// If there was no failure, add the python variables to the platformInfo
-	if e == nil {
-		platformInfo["pythonV"] = pythonV
-	} else {
-		warnings = append(warnings, fmt.Sprintf("could not collect python version: %s", e))
-	}
```

Comment:
Neat: the backend already handle not having `pythonV` so we don't have to do anything else !